CHANGELOG.md

```diff
@@ -15,6 +15,7 @@
 #### :boom: Breaking Change
 
 - Removed "rescript legacy" subcommand in favor of separate "rescript-legacy" binary. https://github.com/rescript-lang/rescript/pull/7928
+- Rewatch: `"type": "dev"` and `dev-dependencies` of local packages will be compiled by default. The `--dev` flag no longer does anything. https://github.com/rescript-lang/rescript/pull/7934
```

Comment:
```suggestion
- Rewatch: `"type": "dev"` and `dev-dependencies` of local packages will be compiled by default for local packages. The `--dev` flag no longer does anything. https://github.com/rescript-lang/rescript/pull/7934
```

Reply:
You just repeat "local packages" here.

Reply:
Ah right, somehow missed the first "local packages". Adapted.